### ISSUE TITLE
Accessibility issue: no focus highlight on forward/backward buttons in image controls

### ISSUE THREAD

**Issue by scoutb-cogapp** [UniversalViewer/universalviewer#1077]
**UV version:** _universalviewer@4.0.25_

**I'm submitting a:** *bug report => please fork one of these [codesandbox examples](https://github.com/UniversalViewer/universalviewer/wiki/UV-Examples) with a repro of your issue and include a link to it below*

### Page area
image controls: forward and backward buttons

### Issue description
The forward and backward buttons inside the image controls (as oppsed to those in the header panel) don't have any focus highlight

### Steps to reproduce
1. open [this manifest](https://www.universalviewer.dev/#?xywh=583%2C1062%2C2492%2C1379&iiifManifestId=https%3A%2F%2Feap.bl.uk%2Farchive-file%2FEAP500-1-1%2Fmanifest&cv=)
2. navigate to the image forward and backward buttons inside the image viewer
3. observe that they have no focus highlight

### Expected behaviour
it is always visually apparent which element on the page currently has keyboard focus

### Possible fix
Turning the divs into actual button or link elements would probagbly automatically fix this

### WCAG criterion
[2.4.7 Focus Visible (Level AA)](https://www.w3.org/WAI/WCAG22/Understanding/focus-visible.html) 

### Related code

```html
<div class="paging btn next" tabindex="0" title="Next" style="position: relative; padding-right: 0px; padding-top: 0px; display: inline-block; top: 239px;"></div>
```
